frames/src/main/java/com/checkout/frames/di/injector/FramesInjector.kt

```diff
@@ -36,13 +39,15 @@ internal class FramesInjector(private val component: FramesDIComponent) : Inject
         internal fun create(
             publicKey: String,
             context: Context,
-            environment: Environment
+            environment: Environment,
+            supportedCardSchemeList: List<CardScheme>? = null
         ): Injector = weakInjector?.get() ?: run {
             val injector = FramesInjector(
                 DaggerFramesDIComponent.builder()
                     .publicKey(publicKey)
                     .context(context)
                     .environment(environment)
+                    .supportedCardSchemes(supportedCardSchemeList)
```

Comment:
It's not a good idea to have a nullable data type in the dependency graph.
I would suggest using non-nullable list and the empty state to replace all current logic that relies on the null state.

Reply:
Improved as discussed